gulpfile.js

```diff
@@ -263,6 +265,7 @@ function dist() {
         './js/vendor/three.js',
         './js/vendor/three.min.js',
         './js/vendor/webm-writer-0.1.1.js',
+        './node_modules/lodash/lodash.min.js',
```

Comment:
This isn't needed. gulp does an `npm install --production` into the `dist/` dir as part of the build, installing all dependencies there.